docs/metrics.md

```diff
@@ -0,0 +1,83 @@
+# Calculated Metrics
+
+These metrics are calculated using stricly `changes`, `deployments` and `incidents` for a rolling window of 90 days.
+
+## Deployment Frequency
+
+Median of the most specific that applies:
+
+| Deployment Frequency | Value                                                    |
+| -------------------- | -------------------------------------------------------- |
+| Yearly               | ???                                                      |
+| Monthly              | One or more calendar week with `deployments`             |
+| Weekly               | One or more `deployments` in a calendar week             |
+| Daily                | Three or more days with `deployments` in a calendar week |
+| On-demand            | ???                                                      |
+
+daysdeploy = number of days with deploys in a certain week (calendar week)
```

Comment:
New lines are missing here

Reply:
I'll remove this when we complete the table above.